status/projects.py

```diff
@@ -278,6 +286,12 @@ class ProjectsSearchHandler(ProjectsBaseDataHandler):
     Loaded through /api/v1/project_search/([^/]*)$
     """
     def get(self, search_string):
+        # Searching for P1234 doesn't work, but 1234 does.
+        # If I'm honest, I don't know why. But let's just strip the P
+        # and get on with enjoying life.
```

Comment:
You shouldn't need to do that. There's a bug in the `search_project_names` function